### PR TITLE
CODE-163

### PR DESCRIPTION
**Pull request by ArBridgeman**

### CODE REVIEW

**Comment by ArBridgeman** [comment]
Would be good to:
- [ ] change the .env password & username; decide if we want  DB name & user to be secrets or not
- [ ] decide if we want to use anisble value or another setup method for our docker-compose secrets
- [ ] check if new docker-compose run configuration ensures that images are updated each time

Would be good to decide if we want other run configurations/helper scripts to:
- [ ] separately run these commands (i.e. instead of docker-compose: 
      python manage.py makemigrations
      && python manage.py migrate
      * when creating a model & making changes in the same branch, how do we ensure that intermediate steps are not unnecessarily saved commits. (As a developer, I want to be able to provide a clean implementation without so many steps)
- [ ] create a backup of the DB; for the dev env, should this be a manual only step (e.g. not hourly) and only hourly on prod?

File: house-keeper/README.md [hunk 1]
```diff
@@ -1,39 +1,32 @@
 # Setup
 
-1. Copy the .env folder into `house_keeper`
+1. Copy the .env folder into `house_keeper/docker`
 2. For storing local database back-ups, create the following folder:
    ```bash
    # for default images (debian)
    sudo mkdir -p /var/opt/pgbackups && sudo chown -R 999:999 /var/opt/pgbackups
    ```
    (Change 999:999 to a valid user:group combination for your system and, if needed, adjust directory permissions.)
```
Comment (ArBridgeman) [inline house-keeper/README.md:9]: @AkagiShigeru Make more specific here

Comment (AkagiShigeru) [reply house-keeper/README.md]: Tried to make more specific.

**Review by AkagiShigeru — APPROVED** [review]
Tested on my mac system that I can execute the ansible-playbook and access the Django instance and also access the running docker instances. Everything that I looked at was working fine.
Also checked that all sous-chef tests (unit + integration) are still working for me locally.
Had a look at the general structure with ansible and the vault and it is looking good and reasonable to me.
@ArBridgeman did we want to add a short playbook to stop the docker instances started with the start-server playbook? If yes, I could try to add this.
Otherwise, it's looking good to me.

File: .idea/runConfigurations/_house_keeper__with_build.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<component name="ProjectRunConfigurationManager">
```
[the remark this answers is not in the document]
Comment (AkagiShigeru) [reply .idea/runConfigurations/_house_keeper__with_build.xml]: Understood yes from talking to Ariel.

**Comment by AkagiShigeru** [comment]
I had to add a minor fix that I had missed before. @ArBridgeman could you please verify that this is ok? My only concern is that the naming 'backup' is generic so that multiple running containers might be returned if a user were running a lot of other things. To make that more defensive we would probably have to be sure that there is no confusion with dashes and underscores etc on our different systems.

**Pull request merged**